meta-balena-common/classes/kernel-balena.bbclass

```diff
@@ -614,6 +613,8 @@ BALENA_CONFIGS[nf_tables] = " \
     CONFIG_NFT_FIB_IPV6=m \
     CONFIG_NF_DUP_IPV6=m \
     "
+BALENA_CONFIGS:append = " ${@configure_from_version("5.9", "", "nf_tables_set", d)}"
```

Comment:
If 5.9 still has this config then shouldn't the above be 5.10 instead of 5.9?

Reply:
you're right @floion I'll address it